src/main/resources/static/ptmexchange/index.html

```
@@ -49,23 +49,24 @@ <h1>Datasets</h1>
       </tr>
 	  <tr class="pc-hov">
-        <td><i>Homo sapiens</i></td><td>Human</td><td>Ubiquitination</td><td>N/A</td><td><a href="https://peptideatlas.org/builds/human/ubiquitination/">Human Ubiquitination 2025-??</td><td>Visible on <a href="https://www.uniprot.org/proteomes/UP000005640">Homo sapiens (Human) proteome<a/></td><td></td><td><a href="./Files/Human_GSB_Ubiq_all_prots_1106.csv.gz">  Results</a></td><td><a href="./Files/2024-04-04-decoys-contam-uniprotkb_proteome_UP000005640.fasta.gz"> Fasta file</a></td>
+        <td><i>Homo sapiens</i></td><td>Human</td><td>Ubiquitination</td><td>N/A</td><td>In progress</td><td>Visible on <a href="https://www.uniprot.org/proteomes/UP000005640">Homo sapiens (Human) proteome</a></td><td></td><td><a href="./Files/Human_GSB_Ubiq_all_prots_1106.csv.gz">  Results</a></td><td><a href="./Files/2024-04-04-decoys-contam-uniprotkb_proteome_UP000005640.fasta.gz"> Fasta file</a></td>
       </tr>
 	  <tr class="pc-hov">
-        <td><i>Homo sapiens</i></td><td>Human</td><td>Acetylation</td><td>N/A</td><td><a href="https://peptideatlas.org/builds/human/acetylation/">Human Acetylation 2025-??</td><td>Visible on <a href="https://www.uniprot.org/proteomes/UP000005640">Homo sapiens (Human) proteome<a/></td><td></td><td><a href="./Files/Human_GSB_Acetyl_all_prots_1106.csv.gz">  Results</a></td><td><a href="./Files/2024-04-04-decoys-contam-uniprotkb_proteome_UP000005640.fasta.gz"> Fasta file</a></td>
+        <td><i>Homo sapiens</i></td><td>Human</td><td>Acetylation</td><td>N/A</td><td>In progress</td><td>Visible on <a href="https://www.uniprot.org/proteomes/UP000005640">Homo sapiens (Human) proteome</a></td><td></td><td><a href="./Files/Human_GSB_Acetyl_all_prots_1106.csv.gz">  Results</a></td><td><a href="./Files/2024-04-04-decoys-contam-uniprotkb_proteome_UP000005640.fasta.gz"> Fasta file</a></td>
       </tr>
 	  <tr class="pc-hov">
-        <td><i>Toxoplasma gondii</i></td><td>Infectious agent of toxoplasmosis</td><td>Phospho</td><td>N/A</td><td><a href="https://peptideatlas.org/builds/toxo/phospho/">T. gondii Phospho 2025&#8209;05</td><td>Visible on <a href="https://www.uniprot.org/proteomes/UP000001529">Toxoplasma gondii (strain ATCC 50611 / Me49) proteome<a/></td><td></td><td><a href="./Files/Tgondii_GSB_phospho_all_prots_0226.csv.gz">  Results</a></td><td><a href="./Files/2024-10-24-decoys-contam-merged_database.fasta.gz"> Fasta file</a></td>
+        <td><i>Toxoplasma gondii</i></td><td>Infectious agent of toxoplasmosis</td><td>Phospho</td><td>N/A</td><td><a href="https://peptideatlas.org/builds/toxo/phospho/">T. gondii Phospho 2025&#8209;05</td><td>Visible on <a href="https://www.uniprot.org/proteomes/UP000001529">Toxoplasma gondii (strain ATCC 50611 / Me49) proteome</a></td><td></td><td><a href="./Files/Tgondii_GSB_phospho_all_prots_0226.csv.gz">  Results</a></td><td><a href="./Files/2024-10-24-decoys-contam-merged_database.fasta.gz"> Fasta file</a></td>
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </a> ], start tag match failed [ <a href="https://peptideatlas.org/builds/toxo/phospho/"> ] on line 61.

(tag-pair)